lib/apis/unsplash.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+const searchPhotos = (pageNo, query) =>
+  axios.get("https://api.unsplash.com/search/photos", {
+    headers: {
+      Authorization: `Client-ID ${process.env.UNSPLASH_API_KEY}`,
+    },
+    params: {
+      page: pageNo,
+      per_page: 30,
+      order_by: "popular",
+      query: query,
+    },
+  });
+
+const unsplashApi = {
+  searchPhotos,
```

Comment:
@labeebklatif 
>We can refrain from this approach and use named exports instead.

This practice is NOT best suited while defining APIs.

Example:
```js
import {searchUnsplash} from "apis/unsplash";
```

^ This naming convention is wrong! We are unnecessarily suffixing with the keyword "Unsplash", even when the API action is already defined within the namespace of "unsplash". Such redundancy can be avoided by using a default import.

Please read: https://www.bigbinary.com/learn-rubyonrails-book/building-and-organizing-apis#building-an-api

@karthiknmenon: Tagging you too over here since I think I had seen a similar review from your end in https://github.com/bigbinary/neeto-kb-web/pull/717#discussion_r779146490. We should stick onto default API imports.

cc: @AbhayVAshokan